networkx/algorithms/isomorphism/tests/test_match_helpers.py

```diff
@@ -62,3 +63,73 @@ def test_generic_multiedge_match(self):
         assert min_flow_match(self.G3[3][4], self.G4[4][5])
         assert not id_match(self.G3[3][4], self.G4[4][5])
         assert not full_match(self.G3[3][4], self.G4[4][5])
+
+
+class TestNumericalMultiEdgeMatch:
```

Comment:
IIUC this is basically a copy of the above test case except using `MultiGraph` instances instead of `MultiDiGraph` instances. Instead of cp/pasting like this, it'd be much better to parametrize the test on the graph type!